src/core_plugins/kibana/public/management/sections/indices/add_data_steps/parse_csv_step/lib/validate_headers.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+export default function validateHeaders(fields) {
+  if (!_.isArray(fields)) {
+    throw new Error('first argument must be an array');
+  }
```

Comment:
I'd remove this guard clause. This isn't really public API for 3rd party clients, where throwing runtime errors can be helpful.

Some clean JSDoc on the function signature would tell the story just as well IMO, and it'd be less code.